README.md

```diff
@@ -6,6 +6,11 @@ For documentation, visit [docs.opentermsarchive.org](https://docs.opentermsarchi
 
 - - -
 
+## Getting Involved
+
+If you want to contribute to the Open Terms Archive Engine, please see [CONTRIBUTING](CONTRIBUTING.md) before sending pull requests.
+You can also report bugs or request features by opening an issue.
```

Comment:
As written in our [CONTRIBUTING.md](https://github.com/OpenTermsArchive/engine/blob/main/CONTRIBUTING.md#copywriting), use neutral or passive wordings.

```suggestion
To contribute to the Open Terms Archive Engine, please refer to the [contributing guidelines](CONTRIBUTING.md) before submitting pull requests. Additionally, you can report bugs or request features by opening an issue.
```

Reply:
Since we want to use “neutral or passive”, shouldn't we go all the way? 🙂 
```suggestion
To contribute to the Open Terms Archive Engine, please refer to the [contributing guidelines](CONTRIBUTING.md) before submitting pull requests. Bugs can be reported or features requested by opening an issue.
```